ReClass.NET/Nodes/ClassNode.cs

```diff
@@ -52,6 +53,52 @@ public static ClassNode Create()
 			return new ClassNode(true);
 		}
 
+
+		/// <summary>
+		/// Initializes the class' name and vtable node from RTTI information, if it's not set already 
+		/// </summary>
+		/// <param name="context"></param>
+		public void InitFromRTTI(DrawContext context)
+		{
+			// first node should be a VTable node or a hex64/32 node
+			if (Nodes.Count <= 0)
+			{
+				return;
+			}
+
+			var rttiInfoFromFirstNode = string.Empty;
+			var firstNode = Nodes[0];
+			if (firstNode is VirtualMethodTableNode vtableNode)
+			{
+				rttiInfoFromFirstNode = vtableNode.GetAssociatedRemoteRuntimeTypeInformation(context);
+			}
+			else
+			{
+				if (firstNode is BaseHexCommentNode baseHexCommentNode)
+				{
+					// ask it as if it might point to a vtable
+					var value = context.Memory.ReadFromBuffer(Offset);
+					rttiInfoFromFirstNode = baseHexCommentNode.GetAssociatedRemoteRuntimeTypeInformation(context, value.IntPtr);
+					if (!string.IsNullOrEmpty(rttiInfoFromFirstNode))
+					{
+						// convert first node to vtable node
+						var newVTableNode = BaseNode.CreateInstanceFromType(typeof(VirtualMethodTableNode));
+						var createdNodes = new List<BaseNode>();
+						this.ReplaceChildNode(firstNode, newVTableNode, ref createdNodes);
+					}
+				}
+			}
```

Comment:
```suggestion
			else if (firstNode is BaseHexCommentNode baseHexCommentNode)
				// ask it as if it might point to a vtable
				var value = context.Memory.ReadFromBuffer(Offset);
				rttiInfoFromFirstNode = baseHexCommentNode.GetAssociatedRemoteRuntimeTypeInformation(context, value.IntPtr);
				if (!string.IsNullOrEmpty(rttiInfoFromFirstNode))
				{
					// convert first node to vtable node
					var newVTableNode = BaseNode.CreateInstanceFromType(typeof(VirtualMethodTableNode));
					var createdNodes = new List<BaseNode>();
					this.ReplaceChildNode(firstNode, newVTableNode, ref createdNodes);
				}
			}
```

Reply:
Removed the braces for the else. Personally not a fan of non-braced if/for blocks but it's your repo so I've removed them :)

Reply:
The missing brace was an (nonfunctional) accident. I want to change the `else { if {` to `else if {`.

Reply:
:) Yeah I figured as much. I think I have corrected it properly now in the last commit :)